### ISSUE TITLE
Lorawan example throws 'ReaderThread' object has no attribute 'read'

### ISSUE THREAD

**Issue by lolsborn** [ronoth/LoStik#15]
Per Bob P.

We ordered a couple of LoSticks from Tindie and tried the example software from the repo, however we cannot get lorawan.py example to work:

    python3 lorawan.py --joinmode otaa --appeui "xxxxxx" --appkey "xxxx" --deveui "xxxxx" /dev/tty.usbserial-14120

Running this on a MacOs with python3 and getting the following error:
```
Connection to LoStik established
mac get appeui
'ReaderThread' object has no attribute 'read'
Lost connection to serial device
Traceback (most recent call last):
File "lorawan.py", line 118, in <module>
with ReaderThread(ser, PrintLines) as protocol:
File "/anaconda3/lib/python3.6/site-packages/serial/threaded/__init__.py", line 250, in __enter__
raise RuntimeError('connection_lost already called')
RuntimeError: connection_lost already called
```
Do you know what I might be doing wrong? (application key supplied in the argument is changed a bit, but functions as example)

Thank you in advance.

Best Regards,
Bob

**Comment by lolsborn** [comment]
This should be fixed again in ffd36ecc15fc95545216bb15b1c78bc6cfa9063d waiting on access to a Gateway this afternoon to test and confirmation from Bob.

**Comment by lolsborn** [comment]
Ok, I've tested the latest commit to master and everything seems to be in order.
